backend/migrations/20220822091356_introduce_endpoints.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE endpoint
+(
+    workspace_id  VARCHAR(50)              NOT NULL REFERENCES workspace (id),
+    path_prefix   varchar(255)             NOT NULL,
+    summary       TEXT                     NOT NULL,
+    description   TEXT                     NOT NULL,
+    target_script varchar(255)             NOT NULL,
+    created_by    VARCHAR(50)              NOT NULL,
+    created_at    TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    archived      BOOLEAN                  NOT NULL DEFAULT false,
+    deleted       BOOLEAN                  NOT NULL DEFAULT false,
```

Comment:
Would replace both with "enabled"

THe point of only allowing archived for script is mostly for tracability which one would have regardless if the endpoint is deleted since the trigger would be done with a permissioned_as